### PR TITLE
Add delete invite action

### PR DESCRIPTION
**Pull request by marceaupatu123**
**Delete Invite**

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [x] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,11 @@
   "devDependencies": {
     "eslint": "^6.8.0",
     "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.22.0",
+    "eslint-config-standard-jsx": "^10.0.0",
```
Comment (DetroitWhiskey136) [inline package.json:13]: This is almost certainly not needed

File: package.json [hunk 2]
```diff
@@ -10,9 +10,11 @@
...
+    "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1"
+    "eslint-plugin-react": "^7.21.5",
```
Comment (DetroitWhiskey136) [inline package.json:17]: This is almost certainly not needed

File: package.json (hunk 1 above)
Comment (marceaupatu123) [reply package.json]: Fixed

File: package.json (hunk 2 above)
Comment (marceaupatu123) [reply package.json]: Fixed

File: actions/delete_invite.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  name: 'Delete Invite',
+  section: 'Invite Control',
+
+  subtitle (data) {
+    return `${data.invite}`
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
```
Comment (almeidx) [inline actions/delete_invite.js:11]:
```suggestion
    if (parseInt(data.storage) !== varType) return
```

File: actions/delete_invite.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const invite = this.evalMessage(data.invite, cache)
+    const reason = this.evalMessage(data.reason, cache)
+    const client = this.getDBM().Bot.bot
+
+    client
+      .fetchInvite(invite)
+      .catch(console.error)
+      .then(async (invite) => {
```
Comment (almeidx) [inline actions/delete_invite.js:42]: Any reason for this to be async?

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/delete_invite.js (hunk 4 above)
Comment (marceaupatu123) [reply actions/delete_invite.js]: Changed.

**Review by almeidx — APPROVED** [review]

**Review by DetroitWhiskey136 — APPROVED** [review]

**Pull request merged**